product_attribute_value_price_formula/models/product_product.py

```diff
@@ -0,0 +1,36 @@
+#  Copyright 2024 Simone Rubino - Aion Tech
+#  License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ProductProduct(models.Model):
+    _inherit = "product.product"
+
+    ptav_formula_extra_price = fields.Float(
```

Comment:
```suggestion
    product_template_attribute_value_formula_extra_price = fields.Float(
```

Explicit is better than implicit

Reply:
Thanks for having a look!
I know, but `ptav` looked like a familiar acronym to me since it's already used throughout Odoo's codebase, for instance https://github.com/odoo/odoo/blob/9e774cf88398b7ffc701c4adf44451e39963db08/addons/website_sale/tests/test_website_sale_reorder_from_portal.py#L45-L47
```
        ptavs = no_variant_template.attribute_line_ids.product_template_value_ids
        ptav_s = ptavs.filtered(lambda ptav: ptav.product_attribute_value_id == s)
        ptav_l = ptavs.filtered(lambda ptav: ptav.product_attribute_value_id == l)
```
Anyhow, I changed it